target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.c

```diff
@@ -34,15 +34,13 @@
 #define RTCL_DDR2		2
 #define RTCL_DDR3		3
 
-#define REG_CTRL0		0
-#define REG_CTRL1		1
-#define REG_COUNT		2
+#define RTCL_REGCNT		2
```

Comment:
Can you put this in the other patch with prefix changes?

And REG_CTRL{0,1} weren't used then?